<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zeitmaschine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>timemachine.svg</normaloff>timemachine.svg</iconset>
  </property>
  <property name="toolTip">
   <string>turn an inbox item into a task or create new task item</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="leCommand"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbAddInboxItem">
              <property name="toolTip">
               <string>turn text to new inbox item</string>
              </property>
              <property name="text">
               <string>add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="zmQTreeWidget" name="twTask">
            <property name="toolTip">
             <string>test</string>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pbClose">
                 <property name="toolTip">
                  <string>mark this item as done</string>
                 </property>
                 <property name="text">
                  <string>done</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="toolTip">
                  <string>dismiss an item</string>
                 </property>
                 <property name="text">
                  <string>dismiss</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbPlusOne">
                 <property name="toolTip">
                  <string>+1 this item</string>
                 </property>
                 <property name="text">
                  <string>+1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="pbMakeAction">
                 <property name="toolTip">
                  <string>turn an inbox item into an action</string>
                 </property>
                 <property name="text">
                  <string>m action</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbMakeProject">
                 <property name="toolTip">
                  <string>turn an inbox item into a project or create new project item</string>
                 </property>
                 <property name="text">
                  <string>m project</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbMakeContext">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>turn an inbox item into context or create new context item</string>
                 </property>
                 <property name="text">
                  <string>m context</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbMakeGroup">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>turn an inbox item into group or create new group item</string>
                 </property>
                 <property name="text">
                  <string>m group</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbMakeNextAction">
                 <property name="text">
                  <string>next action</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QPushButton" name="pbOrder">
                 <property name="toolTip">
                  <string>make zeitmaschine tell you what to do</string>
                 </property>
                 <property name="text">
                  <string>order a task</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>clean inbox</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QPushButton" name="pbDelete">
                 <property name="text">
                  <string>delete</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbSync">
                 <property name="toolTip">
                  <string>sync model with public persistance</string>
                 </property>
                 <property name="text">
                  <string>sync</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QTextEdit" name="teNotes">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <italic>true</italic>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>zmQTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>zmQTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
